ReactNativeClient/lib/models/Note.js

```diff
@@ -279,6 +280,24 @@ class Note extends BaseItem {
 		return results.length ? results[0] : null;
 	}
 
+	static async getUpdatedPreview(noteId) {
+		const preview = KvStore.instance().db_ ? await KvStore.instance().value(noteId) : '';
+		return preview;
+	}
+
+	static previewFromHtml(html) {
```

Comment:
I was trying to extract the preview from the rendered HTML. Will that not work? I don't get exactly how it will fail on other platforms.
Will I need to write a `markdown-it` plugin to extract the text from the markdown?